graphics/unpaper/Portfile

```diff
@@ -2,43 +2,43 @@
 
 PortSystem          1.0
 PortGroup           github 1.0
+PortGroup           meson 1.0
 
-# When updating to the next version, please switch to xz tarball
-# and no longer autoreconf.
-github.setup        Flameeyes unpaper 6.1 unpaper-
-revision            2
+github.setup        unpaper unpaper 7.0.0 unpaper-
+github.tarball_from releases
+use_xz              yes
+
+name                unpaper
+revision            0
 categories          graphics
-maintainers         nomaintainer
-license             GPL-2+
+maintainers         {@akierig fastmail.de:akierig} openmaintainer
+license             GPL-2
 
 description         Post-processing scanned and photocopied book pages
 long_description    unpaper is a post-processing tool for scanned sheets of paper, especially for book pages that have been scanned from previously created photocopies. \
                     The main purpose is to make scanned book pages better readable on screen after conversion to PDF. \
                     Additionally, unpaper might be useful to enhance the quality of scanned pages before performing optical character recognition (OCR).
-platforms           darwin
 
 homepage            https://www.flameeyes.eu/projects/unpaper
 
-checksums           rmd160  29bfd4b927ecd5e56d49b0f8f540d5138544a8a4 \
-                    sha256  9421fa66c8506f12dc0cc4c3814f1b41903420f6afeecd0db50fae49fcdb5740 \
-                    size    2704127
+checksums           rmd160  704cbff899afa1a21268351afbcb76556422844d \
+                    sha256  2575fbbf26c22719d1cb882b59602c9900c7f747118ac130883f63419be46a80 \
+                    size    4430572
+
+set py_ver          3.12
+set py_ver_nodot    312
 
-depends_build       port:docbook-xsl-ns \
+depends_build-append \
+                    port:cmake \
```

Comment:
Both cmake and meson?

Reply:
Was fixed in a version that was uncommitted apparently. fixed.